bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -93,33 +93,33 @@ Equipment,Gate,Door,Gate,Gates,
 Equipment,InnerDoor,Door,Inner Door,Inner Doors,
 Equipment,SideDoor,Door,Side Door,Side Doors,
 Equipment,Doorbell,,Doorbell,Doorbells,
-Equipment,Fan,,Fan,Fans,
+Equipment,Fan,HVAC,Fan,Fans,
 Equipment,CeilingFan,Fan,Ceiling Fan,Ceiling Fans,
 Equipment,KitchenHood,Fan,Kitchen Hood,Kitchen Hoods,
 Equipment,HVAC,,HVAC,"Heating, Ventilation, Air Conditioning, A/C, A/Cs, AC",
-Equipment,Inverter,,Inverter,Inverters,
-Equipment,LawnMower,,Lawn Mower,Lawn Mowers,
-Equipment,Lightbulb,,Lightbulb,"Lightbulbs, Bulb, Bulbs, Lamp, Lamps, Lights, Lighting",
-Equipment,LightStripe,Lightbulb,Light Stripe,Light Stripes,
+Equipment,Inverter,PowerSupply,Inverter,Inverters,
+Equipment,LawnMower,Horticulture,Lawn Mower,Lawn Mowers,
+Equipment,LightSource,,Light Source,"Lightbulbs, Bulb, Bulbs, Lamp, Lamps, Lights, Lighting",
+Equipment,LightStrip,LightSource,Light Strip,"Light Stripe, LED strip",
```

Comment:
```suggestion
Equipment,LightStrip,LightSource,Light Strip,"LightStripe, LED strip",
```

Reply:
If we want to really maintain compatibilities, include `Light Stripes` as synonyms too, although I doubt anyone used this version (`Light Stripes`).

I'm guessing that a lot of people would've been using `LightStripe` though, hence it's important to keep that one as synonym.